GOVERNANCE.md

```diff
@@ -1,137 +1,129 @@
-including:
+The Technical Committee (TC) has final authority over technical issues regarding
+this project, including:
 
 * Technical direction
-* Project governance and process (including this policy)
+* Project governance
 * Contribution policy
 * GitHub repository hosting
-* Conduct guidelines
 * Maintaining the list of additional Collaborators
 
-* [Current list of TSC members](./README.md#current-project-team-members)
+* [Current list of TC members](./README.md#current-project-team-members)
 
-The operations of the TSC are governed by the [TSC Charter][] as approved by
-the Node.js Foundation Board of Directors.
+All TC members are owners of the Ayo.js GitHub organization, and also have admin
+rights in the Ayo.js Discord server.
 
-### TSC Meetings
+## Community Committee
 
-The TSC meets regularly in a voice conference call. The meeting is run by a
-designated meeting chair approved by the TSC. Each meeting is streamed on
-YouTube.
+The Community Committee (CC) is responsible for handling issues regarding the
+broad community of the Ayo.js project. This includes, but is not limited to:
 
-Items are added to the TSC agenda which are considered contentious or
-are modifications of governance, contribution policy, TSC membership,
-or release process.
+* Evangelism (this includes outward appearance, social media, etc)
+* Conferences
+* Meetups
+* Conduct
+* Moderation
 
-The intention of the agenda is not to approve or review all patches.
-That should happen continuously on GitHub and be handled by the larger
-group of Collaborators.
+* [Current list of CC members](./README.md#current-project-team-members)
 
-Any community member or contributor can ask that something be reviewed
-by the TSC by logging a GitHub issue. Any Collaborator, TSC member, or the
-meeting chair can bring the issue to the TSC's attention by applying the
-`tsc-review` label. If consensus-seeking among TSC members fails for a
-particular issue, it may be added to the TSC meeting agenda by adding the
-`tsc-agenda` label.
+All CC members are owners of the Ayo.js GitHub organization, and also have admin
+rights in the Ayo.js Discord server.
 
-Prior to each TSC meeting, the meeting chair will share the agenda with
-members of the TSC. TSC members can also add items to the agenda at the
-beginning of each meeting. The meeting chair and the TSC cannot veto or remove
-items.
+### Correspondence between TC and CC
 
-The TSC may invite additional persons to participate in a non-voting capacity.
+The TC and the CC are normally completely independent from each other, however
+in the event where an issue arises that crosses both of their areas of
+expertise, a joint action is required. In these events, none of the two groups
+may make a final decision until the other agrees.
 
-The meeting chair is responsible for ensuring that minutes are taken and that a
-pull request with the minutes is submitted after the meeting.
+## Elected Final Arbiters
```

Comment:
I know part of this document is to figure out how members get onto each commit so I wanted to start a review thread within this pull to talk about that.

My thoughts (opinions)(perception):

- Members elect EFA's
- Possibly EFA's elect TC and CC members Alternatively TC & CC are elected by community members.
- Upon term expiring for an EFA then a review should kick off to remove or add members to the TC or CC (Similar to a president coming in and out and re-evaluating the cabinet staff?)
- TC and CC should have terms as well, but may be re-elected. This will force a periodic review of the member if we aren't reviewing them via the EFA elections.

Other thoughts? These are just ideas and should be taken as that at face value.